Trappist/src/Promact.Trappist.Test/Questions/QuestionsRepositoryTest.cs

```diff
@@ -13,15 +21,143 @@ namespace Promact.Trappist.Test.Questions
     public class QuestionsRepositoryTest : BaseTest
     {
         private readonly IQuestionRespository _questionRepository;
+        private readonly UserManager<ApplicationUser> _userManager;
 
         public QuestionsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
         {
             //resolve dependency to be used in tests
             _questionRepository = _scope.ServiceProvider.GetService<IQuestionRespository>();
+            _userManager = _scope.ServiceProvider.GetService<UserManager<ApplicationUser>>();
             ClearDatabase.ClearDatabaseAndSeed(_trappistDbContext);
         }
 
         /// <summary>
+        /// Test to add single answer Question
+        /// </summary>
+        [Fact]
+        public async Task AddSingleAnswerQuestionAsync()
+        {
+            var singleAnswerQuestion = await CreateSingleAnswerQuestion();
+            ApplicationUser user = new ApplicationUser() { Email = "vihar@promactinfo.com", UserName = "vihar@promactinfo.com" };
+            await _userManager.CreateAsync(user);
+            var applicationUser = await _userManager.FindByEmailAsync(user.Email);
+            await _questionRepository.AddSingleMultipleAnswerQuestionAsync(singleAnswerQuestion, applicationUser.Id);
+            Assert.True(_trappistDbContext.Question.Count() == 1);
+            Assert.True(_trappistDbContext.SingleMultipleAnswerQuestion.Count() == 1);
+            Assert.True(_trappistDbContext.SingleMultipleAnswerQuestionOption.Count() == 4);
+        }
+
+        /// <summary>
+        /// Test to add multiple answer Question
+        /// </summary>
+        [Fact]
+        public async Task AddMultipleAnswerQuestionAsync()
+        {
+            var multipleAnswerQuestion = await CreateMultipleAnswerQuestion();
+            ApplicationUser user = new ApplicationUser() { Email = "vihar@promactinfo.com", UserName = "vihar@promactinfo.com" };
```

Comment:
the magic string should be used only once. if same magic string occurs more than once then please use a variable for it.